source/best-practices/backup.rst

```diff
@@ -111,6 +112,22 @@ Windows
 robocopy
 ^^^^^^^^
 
+使用 Windows 自带的 `robocopy <https://docs.microsoft.com/zh-cn/windows-server/administration/windows-commands/robocopy>`__
+命令进行增量备份，只同步有改动的文件。假设要备份整个 D 盘，移动硬盘下的备份目录为 :file:`F:\\backup` 。
+使用以下命令可以将 D 盘里的所有子目录和文件完整同步到备份目录下，此时备份目录是 D 盘的一个镜像::
+
+    $ robocopy "D:\ " "F:\backup " /mir /mt
+
+.. important::
+
+   以上命令中目录的引号、最后的斜杠和最后的空格非常重要。
+   若没有斜杠和空格（\ :file:`"D:"` 和 :file:`"F:\\backup"`\ ），备份的是脚本所在目录，并非 D 盘。
```

Comment:
Since we run this command in CMD or PowerShell, I'm not sure if it will backup the current dir.